### ISSUE TITLE
Basic rules; what is the -actual- average number of stat points?

### ISSUE THREAD

**Issue by 1sourcecontrol** [trowl223/Compound_X#113]
Herro, In basic rules (dev branch) it is stated that an average person would have 5 points in each primary stat. 5 * (7 stats) = **35**. While it may appear that the average is 5 because our soft-maximum is 10 points in any primary stat...
The average _human_ would actually have 6 points: 6 * (7 stats) = **42**.  And no known (at least, to me) race has 35 starting points. Also, _Trowl223[EDIT]_ has stated at past CX events that the average quantity of stat points is 6 (so that assumes the average player is human?).
I suggest rewording the relevant lines in the basic rules.
https://github.com/trowl223/Compound_X/blob/c70445a7ef6e253e64a9887b813e102b9dff6123/play/PlayerRules/Basic%20Rules.txt#L134

Also, unrelated, I saw a textual error 
https://github.com/trowl223/Compound_X/blob/c70445a7ef6e253e64a9887b813e102b9dff6123/play/PlayerRules/Basic%20Rules.txt#L204
We do not get a +1 stat point at the same rate as class feats.

**Comment by bleehu** [comment]
The average person in the world would have a '5', but player characters are not average people; they're either heroes, anti-heroes, or supervillians. So while *people* have an average of 5, *PCs* have an average of around 41. 

This is rabbit-trailing a bit, but I think we should slightly up the count of stat points we give out before level 10.

**Comment by 1sourcecontrol** [comment]
Ok, good word; however, the text is completely ambiguous.

No worries, this is a simple topic. Might as well have even more productive conversations.
How come? There are lots of considerations (as I'm sure you are aware).

**Comment by bleehu** [comment]
ya, we could shore up the description a bit.

There are a lot, but I think our conversations will be more productive if we approach each consideration in turn. Which popped out at you?

**Comment by 1sourcecontrol** [comment]
I don't know where the stat point limit should be. Max 15/14 stat points works for me as it currently stands. But I don't have a feel for how the balance may shift if players start having 15/14 stat points around level 10. I'm trying to design weapons and armor for a certain expected miss %, health, and other stats. If the average player starts having much higher perception/Health than I designed for, it may imbalance combat....

Also, can of worms opening, giving players more stat points makes it harder to have usefulness from external augmentation if external aug gives stat points.

Do you have a specific reason in mind for upping the stat count?

**Comment by trowl223** [comment]
Bleehu is correct on all counts. I think 15 is a good cap for now until we do more extensive testing. that grants a +40, which is a whole heck of a lot of points. I'm also thinking lately we should halve all bonuses from attachments, anyone with a +40 perception won't be hindered by any sort of cover.

**Comment by bleehu** [comment]
I think these are good changes, but we should look at how these stats progress over leveling. A particular static maximum I think makes less sense at either early levels or late levels. 
Perhaps we should also add just a little scaling to cover? Perhaps reflecting veterans' tendency to take cover more efficiently having been battle-hardened?

**Comment by trowl223** [comment]
Yeah, hard caps aren't always the best method for balancing. maybe we could look at different caps for Lvls 7, 14 and 20?

I believe @1sourcecontrol is working on that with the new combat tree? If not, It'd make for a good feat tree which rewards our players for playing how we think they should.

Edit:
I had a shower thought this morning. I'm going to toy with different stat / skill increase charts per race starting on the 5th or 6th of July. During this experiment I'll start with a comparison chart of how the current races all level, including the new Rock and Plant races which are as-of-yet unwritten.

**Comment by 1sourcecontrol** [comment]
I'm pretty fine with the idea of a 10-attribute point hard/soft cap at character creation and a 15-point hard cap thereafter. So that a cyborg could get shmancy upgrades to push him over 10 attribute points during his lifetime and/or characters can invest their bonus skill points to push themselves over 10. 
Another thought is to put a soft cap at 8 or 9 points during character creation. Which allows characters with racial traits, that offer bonuses, to push their respective attribute points up to 10 or so. Giving more interesting choices to race picking, If that makes sense.

Also, since M-dawg brought up the:
> scaling cover with player level

I find it appropriate to tell you guys here, in Weapon 2.0 I'm going to propose that (Weapon Ranged Miss Chance) + (PER mod) + (Weapon Attachments) should be lower-bounded at around -10% miss chance. And separately, (Automatic/Burst Miss Chance) + (Relevant Modifiers) should be lower-bounded around 10% miss chance.
Also, I will propose reducing PER mod to a max of -25% Miss Chance at 15 attribute points. That way, PER won't be such an overwhelming modifier in miss chance. (Reason also being, I've nuanced attachments so that _not_ all attachments directly contribute to range accuracy; i.e. bolting on a red-dot, foregrip, and a flash suppressor doesn't give -30% range miss chance, but something in the realm of -10% range miss chance and -20% automatic miss chance). In other words, i'm trying to make/maintain the ways to improve accuracy similar in magnitude so that there won't be the option for a single best-way to improve accuracy.
1. Logic for realism: weapon attachments in real life will have diminishing returns; a real human can never reach perfect accuracy by bolting on every available attachment. Also, a real human can never reach such a high state of perception as to have constant perfect accuracy at targets behind cover at 100 yards.
2. Logic for gameplay: DND example. DND's armor system (Armor Class) provides increasing miss chance with player level. CX offers no such system. Miss chances are static. In other words, increasing player accuracy is not countered by increasing enemy defense in CX; eventually allowing players to achieve perfect combat accuracy if every miss chance variable is not carefully considered in all situations. Adding a cap is the easiest way to mitigate that while allowing easier development of more powerful weapons and abilities (less abuse-checks to do).

**Comment by trowl223** [comment]
Soft capping at 8 or 9 would be ab interesting experiment. It pushes for a more even spread of skills rather than min-maxing and means that players who really want that 10 score at level 1 need to pick a race that gives them a bonus in that score. 
Since we're on the subject of Racial score bonuses, has anyone else noticed that with a point-buy system, the stat that the race's bonus applies to doesn't matter? If you have a Tundarin with +1 Dex and you really want to build a fort heavy character that is a cat, you can just spend extra point in fort and put your Dex 1 point lower than what you want it to be and let the bonus take over. This really ends up as a bonus to your Fort, not Dex. Should I open up a separate issue where we debate this phenomenon?

I think those lower bounds are acceptable.
I agree and would like to add that it makes sense to have abilities be the majority contributing factor to a character's skill with a weapon. Experience should trump hardware.

**Comment by 1sourcecontrol** [comment]
@trowl223 
I suggest we try 9 (which I can propose in the combat 2.0 update.) An even spread is unintentional on my part; I don't think it's ideal to reduce player choice/customization with primary attributes. But it will increase the necessity to pick a race that fits your needs and desires, as you noted. 

I have observed that in the past; the bonus primary attribute points only force a minimum DEX/STR/etc. I don't suggest looking at the situation like: "This really ends up as a bonus to your Fort, not Dex". It's still a bonus to dex because they have a minimum of 2 points in DEX and that +1 is something they wouldn't have had otherwise. And, in your example, if they already were going to build a 10 dex character, the bonus point doesn't _appear_ to have done anything other than give another point for something else but if they were making a 9 dex character, it would be a bonus... However, to make the bonus more meaningful, I think the +1 DEX should allow the Tundarin to have 11 DEX at the start (or 10 if we reduce the normal max to 9).
You can make a separate issue if you like, no harm in that.

Sweet. 
Along the lines of "abilities be the majority contributing factor"  I forgot to mention: even though the normal range miss chance minimum is -10%, etc., I intend for the player to be able to use temporary abilities to exceed the minimum. That way, abilities will (by-in-large) never be useless with accuracy bonuses.

**Comment by 1sourcecontrol** [comment]
made the text clarifications in #118 . Leaving open because of the discussion?

**Comment by bleehu** [comment]
Close it once it's merged! :)
